<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>U2::ExportSequencesDialog</class>
 <widget class="QDialog" name="U2::ExportSequencesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>466</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Export Selected Sequences</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="2" column="0">
      <widget class="QCheckBox" name="withAnnotationsBox">
       <property name="text">
        <string>Export with annotations</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QCheckBox" name="addToProjectBox">
       <property name="text">
        <string>Add document to the project</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="fileLabel">
       <property name="text">
        <string>Export to file</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="fileNameEdit"/>
     </item>
     <item row="0" column="2">
      <widget class="QToolButton" name="fileButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="formatLabel">
       <property name="text">
        <string>File format to use</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QComboBox" name="formatCombo"/>
     </item>
     <item row="4" column="0">
      <widget class="QCheckBox" name="customSeqNameBox">
       <property name="text">
        <string>Use custom sequence name</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="2">
      <widget class="QLineEdit" name="sequenceNameEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="translationBox">
     <property name="title">
      <string>Convertion options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <widget class="QRadioButton" name="directStrandButton">
        <property name="text">
         <string>Save direct strand</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QRadioButton" name="complementStrandButton">
        <property name="text">
         <string>Save complement strand</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QRadioButton" name="bothStrandsButton">
        <property name="text">
         <string>Save both strands</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="translateButton">
        <property name="text">
         <string>Translate to amino alphabet</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="allTFramesButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Save all amino frames</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QComboBox" name="translationTableCombo">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="translationTableButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Use custom translation table</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="backTranslationBox">
     <property name="title">
      <string>Back translation options</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="backTranslateButton">
        <property name="text">
         <string>Translate back to nucleic alphabet</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,0">
        <item>
         <widget class="QRadioButton" name="mostFrequentlyButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Most frequently used codons</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="frequencyDistributionButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Frequency distribution</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
        <item>
         <widget class="QLabel" name="organismLabel">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Organism:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="organismCombo">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="frame">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="saveModeBox">
     <property name="title">
      <string>Merge options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" rowstretch="0,0" columnstretch="0,1">
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="merge_gap_label">
          <property name="text">
           <string>Add gap symbols between sequences</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="mergeSpinBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="maximum">
           <number>100000</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="mergeButton">
        <property name="text">
         <string>Merge sequences</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="separateButton">
        <property name="text">
         <string>Save as separate sequences</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>4</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="exportButton">
       <property name="text">
        <string>Export</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>fileNameEdit</tabstop>
  <tabstop>fileButton</tabstop>
  <tabstop>formatCombo</tabstop>
  <tabstop>withAnnotationsBox</tabstop>
  <tabstop>addToProjectBox</tabstop>
  <tabstop>customSeqNameBox</tabstop>
  <tabstop>sequenceNameEdit</tabstop>
  <tabstop>directStrandButton</tabstop>
  <tabstop>complementStrandButton</tabstop>
  <tabstop>bothStrandsButton</tabstop>
  <tabstop>translateButton</tabstop>
  <tabstop>allTFramesButton</tabstop>
  <tabstop>translationTableButton</tabstop>
  <tabstop>translationTableCombo</tabstop>
  <tabstop>backTranslateButton</tabstop>
  <tabstop>mostFrequentlyButton</tabstop>
  <tabstop>frequencyDistributionButton</tabstop>
  <tabstop>organismCombo</tabstop>
  <tabstop>separateButton</tabstop>
  <tabstop>mergeButton</tabstop>
  <tabstop>mergeSpinBox</tabstop>
  <tabstop>exportButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>U2::ExportSequencesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>528</x>
     <y>456</y>
    </hint>
    <hint type="destinationlabel">
     <x>251</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mergeButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>mergeSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>518</x>
     <y>379</y>
    </hint>
    <hint type="destinationlabel">
     <x>517</x>
     <y>406</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>translateButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>allTFramesButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>184</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>184</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>backTranslateButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>mostFrequentlyButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>132</x>
     <y>269</y>
    </hint>
    <hint type="destinationlabel">
     <x>133</x>
     <y>293</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>backTranslateButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>frequencyDistributionButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>132</x>
     <y>269</y>
    </hint>
    <hint type="destinationlabel">
     <x>517</x>
     <y>293</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>backTranslateButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>organismLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>132</x>
     <y>269</y>
    </hint>
    <hint type="destinationlabel">
     <x>69</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>backTranslateButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>organismCombo</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>132</x>
     <y>269</y>
    </hint>
    <hint type="destinationlabel">
     <x>441</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>translateButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>translationTableButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>120</x>
     <y>184</y>
    </hint>
    <hint type="destinationlabel">
     <x>120</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>customSeqNameBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>sequenceNameEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>161</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>197</x>
     <y>103</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
